test/e2e/adminUI/tests/group005Fields/commonFieldTestUtils.js

```diff
@@ -0,0 +1,128 @@
+module.exports = {
+	beforeUI: function (browser) {
+		browser
+		browser.app = browser.page.app();
+		browser.signinPage = browser.page.signin();
+		browser.listPage = browser.page.list();
+		browser.initialFormPage = browser.page.initialForm();
+
+		browser.app.navigate();
+		browser.app.waitForElementVisible('@signinScreen');
+
+		browser.signinPage.signin();
+		browser.app.waitForElementVisible('@homeScreen');
+	},
+	beforeUX: function (browser) {
+		browser.app = browser.page.app();
+		browser.signinPage = browser.page.signin();
+		browser.listPage = browser.page.list();
+		browser.itemPage = browser.page.item();
+		browser.initialFormPage = browser.page.initialForm();
+
+		browser.app.navigate();
+		browser.app.waitForElementVisible('@signinScreen');
+
+		browser.signinPage.signin();
+		browser.app.waitForElementVisible('@homeScreen');
+	},
+	after: function (browser) {
+		browser.app.signout();
+		browser.end();
+	},
+	assertInitialFormUI: function (name) {
+		var nameLowercase = name.toLowerCase();
+		return function (browser) {
+			browser.app
+				.click('@fieldListsMenu')
+				.waitForElementVisible('@listScreen')
+				.click('@' + nameLowercase + 'ListSubmenu')
+				.waitForElementVisible('@listScreen');
+
+			browser.listPage
+				.click('@createFirstItemButton');
+
+			browser.app
+				.waitForElementVisible('@initialFormScreen');
+
+			browser.initialFormPage.section.form.section[ nameLowercase + 'List' ].section.name
+				.verifyUI();
+
+			browser.initialFormPage.section.form.section[ nameLowercase + 'List' ].section.fieldA
```

Comment:
it's confusing that the function parameter is called `name` and section. `name`  perhaps call the input parameter `listName`?